### ISSUE TITLE
[FEATURE] Connect my HubSpot account via a private HubSpot app

### ISSUE THREAD

**Issue by dannyroosevelt** [PipedreamHQ/pipedream#5023]
I want to be able to provide my own access token from a private HubSpot account and still use all the same built-in actions and triggers for HubSpot on Pipedream.

**Comment by raphaelvarieras** [comment]
Not a problem - I've been using my own private app since they discontinued API keys:

<img width="1296" alt="image" src="https://user-images.githubusercontent.com/6558178/210172968-5c72c628-9505-4b59-90ca-4a734b4cb6c0.png">

Once you have the private app token, just use it the same as the API keys. Headers should look as follows after you've stored the app token in your Pipedream secrets:
```
"Content-Type": "application/json",
Authorization: `Bearer ${process.env.PRIVATE_APP_TOKEN}`,
```

**Comment by dannyroosevelt** [comment]
Thanks @raphaelvarieras. But you can only use your private HS app in code directly, and are not able to use any of the pre-built triggers or actions. Is that not an issue for you?

**Comment by raphaelvarieras** [comment]
Ah yes, fair enough @dannyroosevelt - I am mostly using the custom actions because the basic ones don't cover my current needs. Sorry for misunderstanding the original issue!

**Comment by dannyroosevelt** [comment]
No problem at all -- 2 followup questions for you:
1. Why are you using a private app? More granular control of scopes? Some other reason?
2. What actions / use cases are missing? Happy to get them on the team's backlog.

**Comment by raphaelvarieras** [comment]
> * Why are you using a private app? More granular control of scopes? Some other reason?
Yes, more granular control of the scopes. I don't necessarily want whoever has access to our Pipedream account to have the same capabilities I have as an individual (super admin). 

> * What actions / use cases are missing? Happy to get them on the team's backlog.
The actions I would like to see added are the batch updates of Hubspot entities. My thinking would be that this action would take arrays of values as inputs to then perform updates in batches. This reduces the number of Pipedream invocations and the number of API calls tallied by Hubspot (there is a quota).
